package.json

```diff
@@ -31,7 +31,8 @@
     "umd-shopify-buy:minify": "uglifyjs --compress --mangle -o dist/shopify-buy.umd.min.js -- dist/shopify-buy.umd.js",
     "umd-shopify-buy-polyfilled:build": "rollup -f cjs --config .rollup.config.umd.polyfilled.js src/shopify-polyfilled.js | node scripts/runBabel umd > dist/shopify-buy.umd.polyfilled.js",
     "umd-shopify-buy-polyfilled:minify": "uglifyjs --compress --mangle -o dist/shopify-buy.umd.polyfilled.min.js -- dist/shopify-buy.umd.polyfilled.js",
-    "commonjs:build": "cross-env BABEL_ENV=commonjs VERSION=$(get-project-version --template v{{version}}-{{commit}}) babel --ignore src/polyfilled.js -d lib/ src/",
+    "commonjs:build": "rm -rf lib; cross-env BABEL_ENV=commonjs VERSION=$(get-project-version --template v{{version}}-{{commit}}) babel --ignore src/polyfilled.js -d lib/ src/",
+    "jsnext:build": "rm -rf lib-jsnext; cross-env BABEL_ENV=jsnext VERSION=$(get-project-version --template v{{version}}-{{commit}}) babel --ignore src/polyfilled.js -d lib-jsnext/ src/",
```

Comment:
Optimized bundle size. We do tree shaking. Most current tooling stacks (browserify, web pack, gulp) don't. Removes (or at least centralizes) weird import/export behavior we've seen in other issues. Narrows support footprint because we control much more of what we expose.